README.md

```diff
@@ -4,26 +4,97 @@ Add badges to your Profile and Projects.
+| Vue.js      | <img src="https://img.shields.io/badge/vuejs%20-%2335495e.svg?&style=for-the-badge&logo=vue.js&logoColor=%234FC08D"/>          | `<img src="https://img.shields.io/badge/vuejs%20-%2335495e.svg?&style=for-the-badge&logo=vue.js&logoColor=%234FC08D"/>`          |
+| Angular     | <img src="https://img.shields.io/badge/angular%20-%23DD0031.svg?&style=for-the-badge&logo=angular&logoColor=white"/>           | `<img src="https://img.shields.io/badge/angular%20-%23DD0031.svg?&style=for-the-badge&logo=angular&logoColor=white"/>`           |
+| Angular.js  | <img src="https://img.shields.io/badge/angular-js%20-%23E23237.svg?&style=for-the-badge&logo=angularjs&logoColor=white"/>      | `<img src="https://img.shields.io/badge/angular-js%20-%23E23237.svg?&style=for-the-badge&logo=angularjs&logoColor=white"/>`      |
+| TailwindCSS | <img src="https://img.shields.io/badge/tailwind-css%20-%2338B2AC.svg?&style=for-the-badge&logo=tailwind-css&logoColor=white"/> | `<img src="https://img.shields.io/badge/tailwind-css%20-%2338B2AC.svg?&style=for-the-badge&logo=tailwind-css&logoColor=white"/>` |
+| Bootstrap   | <img src="https://img.shields.io/badge/bootstrap%20-%23563D7C.svg?&style=for-the-badge&logo=bootstrap&logoColor=white"/>       | `<img src="https://img.shields.io/badge/bootstrap%20-%23563D7C.svg?&style=for-the-badge&logo=bootstrap&logoColor=white"/>`       |
+| Material UI | <img src="https://img.shields.io/badge/material%20ui%20-%230081CB.svg?&style=for-the-badge&logo=material-ui&logoColor=white"/> | `<img src="https://img.shields.io/badge/material%20ui%20-%230081CB.svg?&style=for-the-badge&logo=material-ui&logoColor=white"/>` |
+| jQuery      | <img src="https://img.shields.io/badge/jquery%20-%230769AD.svg?&style=for-the-badge&logo=jquery&logoColor=white"/>             | `<img src="https://img.shields.io/badge/jquery%20-%230769AD.svg?&style=for-the-badge&logo=jquery&logoColor=white"/>`             |
+| Django      | <img src="https://img.shields.io/badge/django%20-%23092E20.svg?&style=for-the-badge&logo=django&logoColor=white"/>             | `<img src="https://img.shields.io/badge/django%20-%23092E20.svg?&style=for-the-badge&logo=django&logoColor=white"/>`             |
+| Rails       | <img src="https://img.shields.io/badge/rails%20-%23CC0000.svg?&style=for-the-badge&logo=ruby-on-rails&logoColor=white"/>       | `<img src="https://img.shields.io/badge/rails%20-%23CC0000.svg?&style=for-the-badge&logo=ruby-on-rails&logoColor=white"/>`       |
+| Django      | <img src="https://img.shields.io/badge/django%20-%23092E20.svg?&style=for-the-badge&logo=django&logoColor=white"/>             | `<img src="https://img.shields.io/badge/django%20-%23092E20.svg?&style=for-the-badge&logo=django&logoColor=white"/>`             |
```

Comment:
You have Django duplicated. Kindly take out the extra one.

Reply:
Oh sorry, fixed it!